mainhandler/handlerequests.go

```diff
@@ -80,12 +81,27 @@ func NewMainHandler(config config.IConfig, k8sAPI *k8sinterface.KubernetesApi) *
 		sendReport:             config.EventReceiverURL() != "",
 	}
 	pool, _ := ants.NewPoolWithFunc(config.ConcurrencyWorkers(), func(i interface{}) {
+		logger.L().Debug("ants pool received job", helpers.Interface("job", i))
```

Comment:
Are the logs in this function leffovers from debugging?

Reply:
No, I consciously added these debug logs in cases where we would need to debug them ever again. They are debug-level, so they will not show up during normal operation.